src/forms/textInput.story.jsx

```diff
@@ -2,115 +2,129 @@ import React from 'react';
 import TextInput from './TextInput';
 import Button from './Button';
 import { storiesOf } from '@kadira/storybook';
-import {
-	IntlProvider,
-	FormattedMessage,
-	defineMessages,
-} from 'react-intl';
 
 storiesOf('TextInput', module)
-	.add('default', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		placeholder='Not your email' />))
-	.add('with value', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		value='Phife Dawg'
-		placeholder='Not your email' />))
-	.add('disabled', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		value='Phife Dawg'
-		placeholder='Not your email'
-		disabled />))
-	.add('error state', () => (<TextInput
-		label='Your name'
-		id='fullname'
-		name='name'
-		error='Not so fast. You have an error.'
-		placeholder='Not your email' />))
-	.add('error state formatted', () => {
-		const trn = defineMessages({
-			error: {
-				defaultMessage: 'This error is a formatted message.',
-				id: 'storybook.anError',
-				description: { jira: 'SDS-204' },
-			}
-		});
+	.add('default', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					placeholder='Not your email' />
+			</div>
+		)
+	)
+	.add('with value', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					value='Phife Dawg'
+					placeholder='Not your email' />
+			</div>
+		)
+	)
+	.add('disabled', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					value='Cannot focus'
+					disabled />
+			</div>
+		)
+	)
+	.add('error state', () => (
+			<div className='span--50'>
+				<TextInput
+					label='Your name'
+					id='fullname'
+					name='name'
+					error='Not so fast. You have an error.' />
+			</div>
+		)
+	)
+	.add('error state as element', () => {
 		return (
-			<IntlProvider defaultLocale='en-US' locale='en-US'>
+			<div className='span--50'>
 				<TextInput label='Your name'
 					id='fullname'
 					name='name'
-					error={<FormattedMessage {...trn.error} />}
-					placeholder='Not your email' />
-			</IntlProvider>
+					error={'<span>This error is an element</span>'}
+					placeholder='not your email' />
+			</div>
```

Comment:
So I would still like to test that error (PropType string or element). correctly renders an element.
I took out the FormattedMessage here and put in a span. This is how it looks now though and I wouldn't expect the span to be stringified.
<img width="476" alt="screen shot 2017-07-06 at 4 09 19 pm" src="https://user-images.githubusercontent.com/92090/27930888-390f71a8-6266-11e7-8d73-c1d284caa043.png">